elastic/beats
# Add log_group_name_prefix config option for aws-cloudwatch input

## What does this PR do?

This PR is to add `log_group_name_prefix` config option into `aws-cloudwatch` input to equivalently support wildcard for `log_group_name`.

## Checklist




## How to test this PR locally
With filebeat input below, you should be able to see Filebeat starts `aws-cloudwatch` input to collect logs from all log groups that has names start with `/aws/ecs`.

```
filebeat.inputs:
- type: aws-cloudwatch
  credential_profile_name: elastic-beats
  log_group_name_prefix: /aws/ecs
  region_name: us-east-1
```

## Related issues

- Closes https://github.com/elastic/beats/issues/21528
